### ISSUE TITLE
Setting for MiniImagenet

### ISSUE THREAD

**Issue by hamidkazemi22** [hushell/pmf_cvpr22#19]
Hi,

I would like to express my appreciation for the excellent repository and the well-organized code. However, I have a question regarding the number of episodes specified in the paper for the MiniImagenet dataset, which differs from the default setting in the code. Specifically, the paper mentions 600 episodes for the test set and 2000 episodes for the training set, while the code seems to use the same number of episodes for both sets. Additionally, the default image size in the code is set to 224 for MiniImagenet, whereas the paper states it should be 84. I was wondering if you could provide an example of how to run MiniImagenet using the correct number of episodes and image size. Thank you in advance for your help.

**Comment by hushell** [comment]
Hello, 

Thank you for your interests! And hugely regret for the late reply! 

> the paper mentions 600 episodes for the test set and 2000 episodes for the training set

You are right. The code was not very rigorous. Actually training set has how many episodes doesn't matter as they are randomly generated. 2000 is just a way to specify the number of iterations for an epoch. 600 episodes for testing is important as this is the convention of Mini-ImageNet for reporting statistics. 

> the default image size in the code is set to 224 for MiniImagenet, whereas the paper states it should be 84. 

We actually take 84x84 images and rescale them to 224x224 to accommodate ViT, rather than using the high-res 224x224 from ImageNet. So the results are comparable with other Mini-ImageNet results.
